controllers/apps/cluster_controller.go

```diff
@@ -143,6 +143,8 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			&clusterServiceTransformer{},
 			// handle the restore for cluster
 			&clusterRestoreTransformer{},
+			// rerender parameters after v-scale and h-scale
+			&clusterParametersTransformer{},
```

Comment:
It should be moved out of the cluster controller since the apps package has no knowledge about the parameters and doesn't depend on it too.